modules/instance_template/variables.tf

```diff
@@ -71,6 +71,16 @@ variable "on_host_maintenance" {
   default     = "MIGRATE"
 }
 
+variable "provisioning_model" {
```

Comment:
This variable does not seem to be used since we switch local.provisioning_model?

Reply:
My bad, that was a leftover from a previous implementation, when preemtible is used only 1 valid value for this parameter so moved to a local.

I have edited the title on this PR to fix actual change